src/Oscoin/Node/Trans.hs

```diff
@@ -58,6 +59,7 @@ data Handle c tx st s i = Handle
     { hConfig       :: Config
     , hNodeId       :: i
     , hStateStore   :: StateStore.Handle c st
+    , hStateStore'  :: ContentStore c st IO
```

Comment:
I am fairly sure the answer is "yes", but let me ask anyway: do you plan to clean this up once the mtl-style stuff is completely replaced, so that we don't have these almost duplicate record fields? (i.e. `hStateStore` vs `hStateStore'`).

Reply:
This has been cleaned up.